cmd/hashsum/hashsum.go

```diff
@@ -15,12 +15,14 @@ import (
 
 var (
 	outputBase64 = false
+	downloadFlag = false
 )
 
 func init() {
 	cmd.Root.AddCommand(commandDefinition)
 	cmdFlags := commandDefinition.Flags()
 	flags.BoolVarP(cmdFlags, &outputBase64, "base64", "", outputBase64, "Output base64 encoded hashsum")
+	flags.BoolVarP(cmdFlags, &downloadFlag, "download", "", downloadFlag, "Download files from remote and hash locally instead of using hash from remote")
```

Comment:
Please factor 4 lines inside init() into new `AddCommand()` and call it from `init()`.